scripts/goldpolish-target-extract-seq.py

```diff
@@ -6,13 +6,22 @@
 import btllib
 
 
+class Coordinate:
+    """Represents start and end positions of a subsequence"""
+
+    def __init__(self, coord):
```

Comment:
See suggestion below for avoiding using a tuple of length 2 as arguments for the function